NitroxPatcher/Patches/Dynamic/Player_TriggerInfectionRevealAsync_Patch.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using UnityEngine;
```

Comment:
Many using here can be removed bc they are not needed

Reply:
cleaned and sorted usings